x-pack/legacy/plugins/siem/public/components/and_or_badge/index.tsx

```diff
@@ -38,10 +38,9 @@ RoundedBadge.displayName = 'RoundedBadge';
 export type AndOr = 'and' | 'or';
 
 /** Displays AND / OR in a round badge */
+// Ref: https://github.com/elastic/eui/issues/1655
```

Comment:
Is this ref necessary now that the ts-ignore is no longer needed? Issue is still open, but seems to be resolved in this instance?